packages/smart/contracts/turbo/FuturesMarketFactory.sol

```diff
@@ -71,4 +71,8 @@ contract FuturesMarketFactory is AbstractMarketFactoryV3, Grouped, ManagedByLink
         require(markets[_marketId].active, "market inactive");
         resolveGroupMarket(_marketId, false);
     }
+
+    function getRewardEndTime(uint256 _eventId) public override returns (uint256) {
```

Comment:
Uses _groupId actually. But shouldn't this return something sensible, not zero?

Reply:
Zero reverts the rewards to payout rewards per day over x days.